src/main/java/duke/FileManager.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileManager {
+    private static final String DIR_PATH = "data";
+    private static final String fileName = "duke.txt";
```

Comment:
Remember to change your constants names to all capital letters as part of the convention (i.e. `FILE_NAME`)